content/en/docs/Policy Reports/_index.md

```diff
@@ -5,61 +5,99 @@ description: >
 weight: 60
 ---
 
-Policy reports are Kubernetes Custom Resources, generated and managed automatically by Kyverno, which contain the results of applying matching Kubernetes resources to Kyverno ClusterPolicy or Policy resources. They are created for `validate` and `verifyImages` rules when a resource is matched by one or more rules according to the policy definition. If resources violate multiple rules, there will be multiple entries. When resources are deleted, their entry will be removed from the report. Reports, therefore, always represent the current state of the cluster and do not record historical information.
+Policy reports are Kubernetes Custom Resources, generated and managed automatically by Kyverno, which contain the results of applying matching Kubernetes resources to Kyverno ClusterPolicy or Policy resources.
+They are created for `validate` and `verifyImages` rules when a resource is matched by one or more rules according to the policy definition.
+If resources violate multiple rules, there will be multiple entries.
+When resources are deleted, their entry will be removed from the report.
+Reports, therefore, always represent the current state of the cluster and do not record historical information.
```

Comment:
No new paragraphs per line, please.

Reply:
what do you mean ?

Reply:
It will render on the same line.

Reply:
I know, but it's written as separate lines.

Reply:
And it's bad ?

Reply:
It's mixing authoring styles and produces inconsistency.

Reply:
Fixing...